DOCS.md

```diff
@@ -92,117 +115,225 @@ This messaging will be displayed to the user for several seconds, after which th
 
 ## How to use TruexAdRenderer
 
-### When to show true\[X\]
+### Setup
```

Comment:
Need to mention here that you can just a hosted library instead, a.k.a:

```javascript
 <script src="https://cdn.jsdelivr.net/npm/@truex/ctv-ad-renderer@latest"></script>
 ```

Reply:
done